Model/Sales/TrackTraceHolder.php

```diff
@@ -243,15 +244,14 @@ private function convertDataForCdCountry($magentoTrack)
                 $this->consignment->addItem($myParcelProduct);
             }
         }
-
         $products = $this->getItemsCollectionByShipmentId($magentoTrack->getShipment()->getId());
 
         foreach ($products as $product) {
             $myParcelProduct = (new MyParcelCustomsItem())
                 ->setDescription($product['name'])
                 ->setAmount($product['qty'])
-                ->setWeight($product['weight'] ?: 1)
-                ->setItemValue($product['price'])
+                ->setWeight($product['weight'] * 1000 ?: 1000)
+                ->setItemValue($product['price'] * 100)
```

Comment:
Ik weet dat we dit samen hebben gedaan, maar het zal mooier zijn als er constanten komen die precies beschrijven wat de  `1000` en `100` precies is